web/src/components/EnableSimulatorButton.tsx

```diff
@@ -1,69 +1,149 @@
+import { Icon, InfoIcon, WarningIcon } from "@chakra-ui/icons";
 import {
   Alert,
+  AlertDescription,
   AlertIcon,
   AlertTitle,
+  Box,
   Button,
+  FormControl,
+  FormControlProps,
+  FormLabel,
+  Switch,
+  SwitchProps,
+  Text,
+  Tooltip,
   useBoolean,
   useColorModeValue,
 } from "@chakra-ui/react";
 import * as React from "react";
-import { useRecoilValue, useSetRecoilState } from "recoil";
+import { useRecoilState, useRecoilValue } from "recoil";
 
 import { trackAnalyticsEvent } from "@/analytics";
 import { setSessionController } from "@/data/queries";
 import { connectionConfig, simulatorEnabled } from "@/data/recoil";
 import { useConnectionState, useMountedCallback } from "@/view/hooks/hooks";
 import { useSession } from "@/view/hooks/useSession";
+import { BsInfoCircleFill } from "react-icons/bs";
 
-export const EnableSimulatorButton = () => {
-  const setEnabled = useSetRecoilState(simulatorEnabled);
+export const EnableSimulatorWarning = () => {
+  return (
+    <Alert status="warning" borderRadius="md">
+      <AlertIcon />
+      <Box display="flex" flexDirection="column">
+        <AlertTitle>The simulator is disabled</AlertTitle>
+        <AlertDescription>
+          This application uses a simulator to generate new data like live
+          notifications and subscribers. To experience the full power of
+          real-time digital marketing, please enable the simulator in the nav
+          bar.
+        </AlertDescription>
+      </Box>
+    </Alert>
+  );
+};
+
+export const SimulatorToggler = ({
+  switchProps,
+  containerProps,
+}: {
+  switchProps?: SwitchProps;
+  containerProps?: FormControlProps;
+}) => {
+  const [enabled, setEnabled] = useRecoilState(simulatorEnabled);
   const { session, refresh: refreshSession } = useSession();
   const config = useRecoilValue(connectionConfig);
   const { connected, initialized } = useConnectionState();
+  const [toggling, togglingCtrl] = useBoolean(false);
 
-  const [enabling, enablingCtrl] = useBoolean(false);
   const stopSpinner = useMountedCallback(
```

Comment:
which spinner is this? is it only related to the disabled property of the switcher?